Sources/NIO/EventLoopFuture.swift

```diff
@@ -401,7 +404,7 @@ public final class EventLoopFuture<Value> {
     internal init(_eventLoop eventLoop: EventLoop, value: Result<Value, Error>?, file: StaticString, line: UInt) {
         self.eventLoop = eventLoop
         self._value = value
-        self._isFulfilled = UnsafeEmbeddedAtomic(value: value != nil)
+        self._isFulfilled = _isDebugAssertConfiguration() ? UnsafeEmbeddedAtomic(value: value != nil) : nil
```

Comment:
This should go in a `debugOnly` block.

Reply:
@Lukasa this wouldn't have worked because it's a `let` property and needs to be initialised before `self` is published which it does when you capture it in the `debugOnly` closure.

But it doesn't matter anymore, I killed the whole atomic now, in debug and release as we totally don't need it.